packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -48,13 +49,18 @@ function Wayfinding({ onStartDirections, onBack, location, onSetSize, isActive,
     /** Indicate if the searched route throws errors */
     const [hasError, setHasError] = useState(false);
 
+     /** Indicate if the search has been triggered */
+    const [searchTriggered, setSearchTriggered] = useState(false);
+
     /** Holds search results given from a search field */
     const [searchResults, setSearchResults] = useState([]);
 
     const [destinationLocation, setDestinationLocation] = useState();
+
```

Comment:
My personal preference is to group related states, refs etc. without empty lines between them. I find that to be more readable.

Reply:
It makes sense, let's stick to this pattern 👍